docs/pages/patterns/comments.vue

```diff
@@ -95,13 +95,69 @@
 
 // Delete Payload
 {
-	id: Number
-	message: String
-	parentId: Number
 }
 
 ```
 
+## Parsing HTML 
+
+KtComment will escape all tags by default but you can opt out and pass your own parser by using the parser prop 
+
+> Remember to still escape tags to prevent XHR attacks,
+you can use KtComment's default parser function with KtComment.defaultParser
+
+```js
+methods: {
+	dangerDefaultParserOverride: msg => escape(msg).replace(/\n/g, '<br>'),
```

Comment:
Should be `dangerouslyOverrideParser`
```suggestion
	dangerouslyOverrideParser: msg => escape(msg).replace(/\n/g, '<br>'),
```